CMakeLists.txt

```
@@ -76,7 +76,6 @@ add_subdirectory(probability)
 add_subdirectory(data_structures)
 add_subdirectory(machine_learning)
 add_subdirectory(numerical_methods)
```

```suggestion
add_subdirectory(numerical_methods)

```
There was a one line space between the two. I have that for readability. This will ensure there are no merge conflicts.